mcr_perception_selectors/ros/launch/object_selector.launch

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
 <launch>
-  <group ns="mcr_perception">
+    <arg name="input_topic" default="/mcr_perception/object_detector/object_list" />
```

Comment:
this argument doesn't seem to be used

Reply:
okay, removed. I added this because this was being passed by planning_bringup, so it was giving error as there was no input_topic param in this launch file.